data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -23,22 +24,35 @@ const apiserver = new kc.Client({
 });
 
 function buildErrors(err) {
```

Comment:
shouldnt the method be named `buildErrorAndThrow` instead? or even better would be `convertToHttpErrorAndThrow(err)`
the action of just `build` would generally mean you return the same error and not throw.

Reply:
Done